src/auth/ui/vue/show.ts

```diff
@@ -188,16 +184,19 @@ export class AuthWebview extends VueWebview {
         return this.ssoSetup(setupFunc)
     }
 
-    private async ssoSetup(setupFunc: () => Promise<any>) {
+    private async ssoSetup(setupFunc: () => Promise<any>): Promise<string> {
         try {
             await setupFunc()
+            return ''
```

Comment:
Can't remember now if this was a problem with the webview impl. but was something weird happening when returning `void`/`undefined`?

Reply:
In this case with it being a string this isn't an issue. It was in the scenario where an arg would be like:

```
{ a: 'A', b: undefined }
```

and we would only get

```
{ a: 'A' }
```

Reply:
Oh that's because everything is serialized between the extension and the webview. `undefined` has no JSON representation and so is equivalent to the absence of a field.